cmd/controller-manager/app/options/options.go

```diff
@@ -50,11 +51,31 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 
 	fs.BoolVar(&o.LimitedScope, "limited-scope", false, "Whether the federation namespace will be the only target for federation.")
 	fs.DurationVar(&o.ClusterMonitorPeriod, "cluster-monitor-period", time.Second*40, "How often to monitor the cluster health")
+
+	fs.BoolVar(&o.LeaderElection.LeaderElect, "leader-elect", true, "Start a leader election client and gain leadership before "+
```

Comment:
(No action required) The addition of so many new configuration parameters suggests to me that implementing component configuration (#114) should be a priority for beta (not for this PR though!).  Thoughts?

Reply:
Agree.. we should definitely add it for beta